OSDOCS#17844: Document replacing a healthy etcd member (#112025)
openshift/openshift-docs: backup_and_restore/control_plane_backup_and_restore/replacing-healthy-etcd-member.adoc

include::_attributes/common-attributes.adoc[]
= Replacing a healthy etcd member
:context: replace-healthy-etcd-member

toc::[]

[role="_abstract"]
You might need to replace a healthy etcd member for planned hardware maintenance, hardware upgrades, or migration to new infrastructure.

include::modules/replace-healthy-etcd-about.adoc[leveloffset=+1]

include::modules/replace-healthy-etcd-procedures-about.adoc[leveloffset=+1]

include::modules/determining-replace-healthy-etcd-member.adoc[leveloffset=+1]

include::modules/replacing-healthy-etcd-member-cpms.adoc[leveloffset=+1]

include::modules/replacing-healthy-etcd-member-machine-api.adoc[leveloffset=+1]

include::modules/replacing-healthy-etcd-member-scale-up-down.adoc[leveloffset=+1]

[role="_additional-resources"]
[id="additional-resources_replace-healthy-etcd-member"]
== Additional resources

* xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backing-up-etcd-data_backup-etcd[Backing up etcd data]
* xref:../../backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.adoc#replacing-unhealthy-etcd-member[Replacing an unhealthy etcd member]

> * xref:../../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[Restoring to a previous cluster state]

Reviewer: ```suggestion
* xref:../../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[Restoring to an earlier cluster state]
```
Tiny edit to match the title of the page that opens

* link:https://docs.redhat.com/en/documentation/assisted_installer_for_openshift_container_platform/2026/html/installing_openshift_container_platform_with_the_assisted_installer/expanding-the-cluster#installing-control-plane-node-healthy-cluster_expanding-the-cluster[Replacing a control plane node in a healthy cluster]
* xref:../../machine_management/deleting-machine.adoc#machine-lifecycle-hook-deletion-etcd_deleting-machine[Quorum protection with machine lifecycle hooks]
* xref:../../machine_management/control_plane_machine_management/cpmso-managing-machines.adoc#cpmso-feat-replace_cpmso-managing-machines[Replacing a control plane machine]
* xref:../../machine_management/control_plane_machine_management/cpmso-manually-scaling-control-planes.adoc#creating-control-plane-node_cpmso-manually-scaling-control-planes[Adding a control plane node to your cluster]

> * link:https://access.redhat.com/articles/6270901[Scaling up and scaling down control plane nodes (Red{nbsp}Hat Knowledgebase article)]

Reviewer: ```suggestion
* link:https://access.redhat.com/articles/6270901[How to replace all master nodes in OpenShift Container Platform 4 (Red{nbsp}Hat Knowledgebase article)]
```
Edit to match the title of the article
